unittests/test_deprecation.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 Insight Software Consortium.
+# Distributed under the Boost Software License, Version 1.0.
+# See http://www.boost.org/LICENSE_1_0.txt
+
+import unittest
+
+from . import parser_test_case
+
+from pygccxml import parser
+from pygccxml import declarations
+
+
+class Test(parser_test_case.parser_test_case_t):
+    global_ns = None
+
+    def __init__(self, *args):
+        parser_test_case.parser_test_case_t.__init__(self, *args)
+        self.header = "test_deprecation.hpp"
+        self.global_ns = None
+        self.config.castxml_epic_version = 1
+
+    def _check_text_content(self, desired_text, comment_decl_string):
+        if comment_decl_string:
+            self.assertEqual(list, comment_decl_string)
```

Comment:
Should be
```suggestion
            self.assertEqual(desired_text, comment_decl_string)
```

Also, how is this passing CI currently? Comparing the Python token `list` with `comment_decl_string`

Reply:
It's likely due to the fact that the check function is getting an empty string to check against in the version of CastXML that is being used.  The ``if`` is there to avoid erroring if the test is run against an older version.  

I can (and will) update the version that is being downloaded in the CI setup.  I also might spend some time making sure those ``print`` messages are actually shown to the CI output like some of the other tests have.

Reply:
@josephsnyder Feel free to re-request a review once the print messages show up. Thanks for updating the CI.